dvc/command/machine.py

```diff
@@ -104,6 +104,53 @@ def run(self):
         return 0
 
 
+class CmdMachineRename(CmdBase):
+    def _check_exists(self, conf):
```

Comment:
Didn't use multi inheritance here, because it is hard to maintain and we only need this `_check_exists` from `CmdMachineConfig` here.